homeassistant/components/modbus/base_platform.py

```diff
@@ -82,6 +83,7 @@ def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
         self._cancel_timer: Callable[[], None] | None = None
         self._cancel_call: Callable[[], None] | None = None
 
+        self._attr_unique_id = entry.get(CONF_UNIQUE_ID, None)
```

Comment:
```suggestion
        self._attr_unique_id = entry.get(CONF_UNIQUE_ID)
```

Reply:
without the default value (None), I'm afraid the unique_id option will become mandatory. 

I test tonight

Reply:
> without the default value (None), I'm afraid the unique_id option will become mandatory.

That statement is incorrect.

Reply:
it's ok